modules/reporters/qmstr-reporter-html/share/skeleton/layouts/shortcodes/copyrights.html

```diff
@@ -0,0 +1,7 @@
+{{ $data := index .Site.Data .Page.Params.project .Page.Params.package .Page.Params.version "data" }} 
+<h3>Copyright</h3>
```

Comment:
Check first that Copyright text exists otherwise don't print these data.
Copyright text is detected only from the spdx-analyzer which is used only in the Calculator project. 
In the other projects the Copyright section will always be empty.